PWGHF/vertexingHF/AliAnalysisTaskSELc2pKs0fromKFP.h

```diff
@@ -0,0 +1,124 @@
+#ifndef ALIANALYSISTASKSELC2PKS0FROMKFP_H
+#define ALIANALYSISTASKSELC2PKS0FROMKFP_H
+
+/// \class AliAnalysisTaskSELc2pKs0fromKFP
+/// \brief This is a brief description of my class
+///
+/// This is a longer description of the class. This longer description is
+/// formatted using the Markdown syntax (see below) and can span on multiple
+/// lines.
+///
+/// \author Jianhui Zhu <zjh@mail.ccnu.edu.cn>, Central China Normal University & GSI Helmholtz Centre for Heavy Ion Research
+/// \date Apr 26, 2019
+
+/* $Id$ */
+
+#ifndef HomogeneousField
+#define HomogeneousField
+#endif
+
+#include "AliAnalysisTaskSE.h"
+#include "AliAODMCParticle.h"
+#include "AliRDHFCutsKFP.h"
+#include "AliNormalizationCounter.h"
+#include "THnSparse.h"
+#include "AliPIDResponse.h"
+#include "AliAODInputHandler.h"
+
+// includes added to play with KFParticle
+#include <vector>
+#include "KFParticleBase.h"
+#include "KFParticle.h"
+#include "KFPTrack.h"
+#include "KFPVertex.h"
+#include "KFVertex.h"
+
+class AliAnalysisTaskSELc2pKs0fromKFP : public AliAnalysisTaskSE  
+{
+    public:
+                                AliAnalysisTaskSELc2pKs0fromKFP();
+                                AliAnalysisTaskSELc2pKs0fromKFP(const char *name, AliRDHFCutsKFP* cuts);
+        virtual                 ~AliAnalysisTaskSELc2pKs0fromKFP();
+
+        virtual void            UserCreateOutputObjects();
+        virtual void            Init();
+        virtual void            LocalInit() {Init();} 
+        virtual void            UserExec(Option_t* option);
+        virtual void            Terminate(Option_t* option);
+
+        void                    SetMC(Bool_t IsMC) {fIsMC=IsMC;}
+        void                    SelectTrack(AliVEvent *event, Int_t trkEntries, Int_t &nSeleTrks, Bool_t *seleFlags);
+        Bool_t                  MakeMCAnalysis(TClonesArray *mcArray);
+        void                    MakeAnaLcFromCascadeHF(TClonesArray *arrayLc2pKs0, AliAODEvent *aodEvent, TClonesArray *mcArray, KFParticle PV);
+        Double_t                InvMassV0atPV(AliAODTrack *trk1, AliAODTrack *trk2, Int_t pdg1, Int_t pdg2);
+        Double_t                CosPointingAngleKF(KFParticle kfp, KFParticle kfpmother);
```

Comment:
You could consider to move these functions (from line 54 to 63) in [AliVertexingHFUtils.h](https://github.com/alisw/AliPhysics/blob/master/PWGHF/vertexingHF/AliVertexingHFUtils.h) to avoid code duplication, since if I am not wrong they are the same as in [AliAnalysisTaskSEXicZero2XiPifromKFP.h](https://github.com/alisw/AliPhysics/blob/796a24bf2ab0dfacd18eb8151ab481466d6dd101/PWGHF/vertexingHF/AliAnalysisTaskSEXicZero2XiPifromKFP.h)

Reply:
Thanks Fabrizio, I will move these functions to AliVertexingHFUtils.h.